### PR TITLE
[FIX] point_of_sale: prevent multi order on single table

### PR DESCRIPTION
**Pull request by sjai-odoo**
Steps to reproduce :
-------------------------
- Install pos_restaurant and open the session.
- Click on a table to create/open an order in that table.
- Click 'Orders' from the burger menu at top right.
- Now in ticketScreen has 'New Order' button at the top left.
- Click the 'New Order' button.

Issue :
---------
A new order will be created on the same table from which we gone to the ticketscreen.

Cause :
----------
We have a function that create an order on the table which is selected so creating an order on the same table.

Fix :
-------
Removed the new order button with the functionality as can be handled by floating orders more cleanly.

task: 4111323

### CODE REVIEW

**Review by caburj — CHANGES_REQUESTED** [review]
Find a way to adapt the tests without adding classes to the components.

File: addons/point_of_sale/static/src/app/components/order_tabs/order_tabs.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
             <t t-set="order" t-value="scope.item" />
             <button t-esc="order.getName()"
                 t-attf-class="{{pos.get_order()?.id === order.id ? 'btn-secondary active' : ''}}"
-                class="btn btn-lg btn-secondary text-truncate mx-1"
+                class="floating-order btn btn-lg btn-secondary text-truncate mx-1"
```
Comment (caburj) [inline addons/point_of_sale/static/src/app/components/order_tabs/order_tabs.xml:13]:
```suggestion
                class="order-tab-button btn btn-lg btn-secondary text-truncate mx-1"
```

File: addons/point_of_sale/static/src/app/generic_components/list_container/list_container.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ export class ListContainer extends Component {
     };
     static template = xml`
         <div class="overflow-hidden d-flex flex-grow-1" t-attf-class="{{props.class}}">
-            <button t-if="props.onClickPlus" class="btn btn-secondary btn-lg me-1" t-on-click="props.onClickPlus">
+            <button t-if="props.onClickPlus" class="new-fo btn btn-secondary btn-lg me-1" t-on-click="props.onClickPlus">
```
Comment (caburj) [inline addons/point_of_sale/static/src/app/generic_components/list_container/list_container.js:37]:
```suggestion
            <button t-if="props.onClickPlus" class="list-plus-btn btn btn-secondary btn-lg me-1" t-on-click="props.onClickPlus">
```

**Comment by sjai-odoo** [comment]
Hi @caburj ,
I have changed it accordingly can you please check it again.
Thanks

**Review by caburj — APPROVED** [review]
robodoo r+

**Pull request closed without merging**